07-errors.Rmd

```diff
@@ -0,0 +1,84 @@
+# Error messages
+
+(This guide is almost entirely asprirational; most of the bad examples come from exiting tidyverse code.)
+
+## "Must"
+
+If you can state what the user should have done to avoid the problem, start with a general statement of what was expected. Frame it possitively, and use "must". When possible, follow it up with a concise description of what was actually supplied:
```

Comment:
I'd break that sentence up, and also add a comma after if you can, to denote conditional for stating what the user should have done vs. starting with the gen'l statement.
"If you can, state what the user should have done. Start with a general statement of what was expected."